js/tools/randAlpha.js

```diff
@@ -0,0 +1,44 @@
+/* global app */
+window.tools.randAlpha = {
+  name: 'randAlpha',
+  icon: '/images/randAlpha.png',
+  state: {
+    selected: false,
+    mousePresesd: false
+    // other state properties go here
+  },
+  events: {
+    mousedown: function (e, self) {
+      self.state.mousePressed = true
+    },
+    mouseup: function (e, self) {
+      self.state.mousePressed = false
+    },
+    mousemove: function (e, self) {
+      // if self tool is selected AND the mouse is pressed
+      if (self.state.selected && self.state.mousePressed) {
+        app.ctx.save()
+        const mouse = app.eventToMouse(e)
+        const fonts = ['Times New Roman', 'Arial', 'monospace', 'Comic Sans', 'fantasy']
+        let randFont = Math.floor(Math.random() * fonts.length)
+        let randSize = Math.floor(Math.random() * 70) + 15
```

Comment:
these two variables are never re-assigned to any other value, which means they should be created with the `const` keyword instead of `let` for better memory management. do u have the StandardJS linter running in ur VSCode? it should have caught this?